salvo/src/lib/benchmark/BUILD

```diff
@@ -27,14 +31,30 @@ py_test(
   ],
 )
 
+py_test(
+  name = "test_scavenging_benchmark",
```

Comment:
So one though, I wonder if someone could come up with  a better term in favor of "scaffolding". Scavenging doesn't sound very uplifting and it is has been my bad to pick that term. No need to change it here as far as I am concerned, but maybe if we can come up with something better and feel it is worth it, we can do a global replacement.

Reply:
Perhaps:
- Discovering Benchmark
- Probing Benchmark
- Foraging Benchmark

The last entry makes me think of bears searching for food, so that may be a something to guide the new choice for a name.

Reply:
I am assuming that if we are renaming, we are going to do it in a separate PR and proceeding with this one as is. Please let me know if that's not the case.

Reply:
@mum4k You are correct.  It would be done in a separate PR.  If I understand @oschaaf correctly, the rename would be universal, not just in Salvo, but NightHawk as well.

Reply:
If we can pick a name everyone is on happier with then I’d be up for creating PRs to replace on both sides

Reply:
@oschaaf did we choose scavenging initially because of the property where the benchmark searches for all the test files / targets to execute? Or is the word scavenging describing some other property of the benchmark?

Reply:
> @oschaaf did we choose scavenging initially because of the property where the benchmark searches for all the test files / targets to execute? Or is the word scavenging describing some other property of the benchmark?

The property this was based on is that it searches/discovers test targets

Reply:
Thank you for confirming that @oschaaf.

If that is the case, it feels like neither of the proposed names explain it well, since it isn't clear **what** is being discovered or probed. How about we go with something more verbose, e.g. `discover_and_run_benchmarks` or similar.

Reply:
`discover_and_run_benchmarks` works for me. It's a bit long but as this  would be spot on in describing what will happen, I think it won't be hard to remember.